src/library/scala/concurrent/ExecutionContext.scala

```diff
@@ -54,8 +54,7 @@ import scala.annotation.implicitNotFound
  * Application callback execution can be configured separately.
  */
 @implicitNotFound("""Cannot find an implicit ExecutionContext. You might pass
-an (implicit ec: ExecutionContext) parameter to your method
-or import scala.concurrent.ExecutionContext.Implicits.global.""")
+an (implicit ec: ExecutionContext) parameter to your method.""")
```

Comment:
You might pass an `(implicit ec: ExecutionContext)` argument to your method or import one in the current scope; make sure choose the `(implicit ec: ExecutionContext)` wisely which matters a lot.